recipes/cocotb/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "cocotb" %}
+{% set version = "1.7.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cocotb-{{ version }}.tar.gz
+  sha256: 959892eb94bd0b3ff40e0fca51d33a3936416deb853e2bac4f7f766b40002650 
+  patches:
+    - patches/0001-remove-static-stdc.patch
+
+build:
+  number: 0
+  skip: true  # [win or py<310]
+  entry_points:
+    - cocotb-config = cocotb.config:main
+  run_exports:
+    - {{ pin_subpackage("cocotb", max_pin="x") }}
+  script:
+    - {{ PYTHON }} -m pip install -U --no-deps . -vvv
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - find-libpython
+  run_constrained:
+    - iverilog >=10.3
+    - verilator >=5.004
+
+test:
+  imports:
+    - cocotb
```

Comment:
You are also installing a python module called "pygpi", but that module is not tested here. Is the missing test a mistake or is the install a mistake?

Reply:
it comes with `cocotb`, I can add a simple import test?